### PR TITLE
README update, Android Studio Emulator

### PR DESCRIPTION
**Pull request by simon-xisystems**
REAME updated to include Android Studio Emulator note.

### CODE REVIEW

**Review by JonasVautherin — CHANGES_REQUESTED** [review]
Thanks a lot! However, that's only for `MavsdkServer` (which is the Android library embedding `mavsdk_server`, the gRPC backend). MAVSDK-Java is only a client and works from the emulator, e.g. you could connect it to a remote `mavsdk_server` running on your computer (or on another computer).

**Comment by simon-xisystems** [comment]
Good point, wording tweaked!

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ It is organized as follows:
 
 The fastest way to start is to follow the instructions in the README of the [java-client](./examples/java-client) example. For Android, the [android-client](./example/android-client) is the next step.
 
-Please note MAVSDK-Java currently is not compatible for running on the Android Studio Emulator. It is only compatible with Arm images. 
+Please note MavsdkServer Java currently is not compatible for running on x86 and x86_64 Android images. if you would like to develop using the MavsdkServer, you will need to deploy and develop using a Arm based product or emulator.
```
Comment (JonasVautherin) [inline README.md:15]:
```suggestion
Please note `MavsdkServer` currently is not compatible for running on x86 and x86_64 Android images. If you would like to develop using `MavsdkServer`, you will need to deploy and develop using an ARM-based product or emulator.
```

**Review by simon-xisystems — COMMENTED** [review]
fab.

**Review by JonasVautherin — APPROVED** [review]
Thanks!

**Pull request merged**